### ISSUE TITLE
Remove the message after a certain time

### ISSUE THREAD

**Issue by sylveon** [rebornix/vscode-encourage#2]
Currently, the message will stay until the next save.

I would prefer if it automatically disappears after a certain time (~10 seconds)

**Comment by rebornix** [comment]
This is a valid request. VS Code Information popup doesn't support auto-dismiss, that's why I moved the encouragement to status bar, then adding auto-dismiss to status bar is must-need.

**Comment by sylveon** [comment]
Something as simple as adding the following code to `updateEncourageMessage` could work.  

``` javascript
setTimeout(10000, function() {
    this ._statusBarItem.hide();
});
```

**Comment by rebornix** [comment]
@charlesmilette already fixed it in 0.0.4 :)
